code/modules/spells/targeted/hypnotise.dm

```diff
@@ -1,6 +1,6 @@
 /spell/targeted/hypnotise
 	name = "Hypnotise (10)"
-	desc = "A piercing stare that incapacitates your victim for a good length of time."
+	desc = "A piercing stare that paralyzes your victim for a good length of time."
```

Comment:
also this is stupid
i wouldn't call "spessman is unable to see, hear, speak, or do anything for 60 seconds" paralysis